### PR TITLE
Added new schema for kafka-to-nexus status and statistics

### PR DESCRIPTION
**Pull request by michele-brambilla**
- [x]  If there are new schema in this PR I have added them to the list in README.md

### CODE REVIEW

File: schemas/fws0_fwr_status.fbs [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+file_identifier "fws0";
+
+struct Statistics {
+avg : float;
+std : float;
+}
+
+table StreamerInfo {
+topic : string;
```
Comment (matthew-d-jones) [inline schemas/fws0_fwr_status.fbs]: Could you indent the fields please?

File: schemas/fws0_fwr_status.fbs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+message_size : Statistics;
+frequency : Statistics;
+}
+
+table StatusInfo {
+streammaster_status : int;
+streamer : [StreamerInfo];
+}
+
+root_type StatusInfo;
```
Comment (matthew-d-jones) [inline schemas/fws0_fwr_status.fbs]: Missing new line at the end of the file.

File: schemas/fws0_fwr_status.fbs [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+struct Statistics {
+avg : float;
+std : float;
+}
+
+table StreamerInfo {
+topic : string;
+messages : int;
+bytes : int;
```
Comment (matthew-d-jones) [inline schemas/fws0_fwr_status.fbs]: Could you make the name a little more specific or add a comment at the end of the line? What is `bytes` the size of, and is it cumulative?

File: schemas/fws0_fwr_status.fbs [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+errors : int;
+message_size : Statistics;
+frequency : Statistics;
```
Comment (matthew-d-jones) [inline schemas/fws0_fwr_status.fbs]: See above.

File: schemas/fws0_fwr_status.fbs [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+table StatusInfo {
+streammaster_status : int;
```
Comment (matthew-d-jones) [inline schemas/fws0_fwr_status.fbs]: What are the allowed values and what do they mean?

File: schemas/fws0_fwr_status.fbs [hunk 6]
```diff
@@ -0,0 +1,24 @@
+// Schema for Kafka-to-NeXus Streamers status
+
+file_identifier "fws0";
+
+struct Statistics {
+avg : float;
+std : float;
```
Comment (matthew-d-jones) [inline schemas/fws0_fwr_status.fbs]: There's no advantage to using `avg` and `std` abbreviations here. `std` is particularly unclear (standard deviation, standard error?).

**Comment by zjttoefs** [comment]
This looks stale. Closing.

**Pull request closed without merging**